### ISSUE TITLE
add other metric ?

### ISSUE THREAD

**Issue by Fab16BSB** [Fab16BSB/Keras-Callback-Realtime-Learning-Curve#4]
Add learning curve with precision, recall and f1-score for the validation set.
May be for the training to ?
